v2/api/network/v1api20220401/storage/groupversion_info_gen.go

```diff
@@ -5,12 +5,13 @@ Licensed under the MIT license.
 
 // Code generated by azure-service-operator-codegen. DO NOT EDIT.
 
-// Package v1api20220401storage contains API Schema definitions for the network v1api20220401storage API group
```

Comment:
minor: Should these comments continue to specify the version in them?

Reply:
I'll tweak to use the folder path in a followup PR:

``` go
// Package v1api20220401/storage contains API Schema definitions for the network v1api20220401/storage API group
```
or something similar.